OpenImis.ModulesV3/ClaimModule/Logic/ClaimLogic.cs

```diff
@@ -22,13 +22,70 @@ public ClaimLogic(IConfiguration configuration, IHostingEnvironment hostingEnvir
             claimRepository = new ClaimRepository(_configuration, _hostingEnvironment);
         }
 
-        public int Create(Claim claim)
+        public List<SubmitClaimResponse> Create(Claim claim)
```

Comment:
Please change the input from Claim to a list of claims: List<Claim> or Claim[]. Then you submit Claim by Claim and get the responses.